src/main/java/com/google/javascript/gents/ModuleConversionPass.java

```diff
@@ -31,10 +38,13 @@
 
   private final Map<String, FileModule> fileToModule;
   private final Map<String, FileModule> namespaceToModule;
+  private final Set<ExportedSymbol> directlyExportedSymbols = new HashSet<>();
+  private final Map<ExportedSymbol, Node> exportsToNodes = new HashMap<>();
```

Comment:
to what kind of nodes? To the actual `export ...`, or to the node that defines the symbol? Or are those identical here? Please document.